mlir/lib/Dialect/MIOpen/Transforms/AffixTuningParameters.cpp

```diff
@@ -46,84 +46,14 @@ struct AffixTuningParameters : public MIOpenOpsAffixTuningParametersPassBase<Aff
-  //           index_t ABlockCopyDstDataPerWrite_M,              - matrix_a_dest_data_per_write_dim_m attribute
-  //           typename BBlockCopyThreadSliceLengths_K_N,        - 
-  //                                                               - GemmBBlockCopyThreadSliceLengths_GemmK_GemmN in GridwiseConvolutionImplicitGemm_v4r4_nchw_kcyx_nkhw
-  //                                                               - Sequence<GemmBBlockCopyThreadSliceLengths_GemmK, GemmBBlockCopyThreadSliceLengths_GemmN>
-  //                                                                 - GemmBBlockCopyThreadSliceLengths_GemmK =
-  //                                                                   k_per_block / GemmBBlockCopyClusterLengths_GemmK
-  //
-  //                                                                 - GemmBBlockCopyThreadSliceLengths_GemmN =
-  //                                                                   n_per_block / GemmBBlockCopyClusterLengths_GemmN
-  //
-  //           typename BBlockCopyThreadClusterLengths_K_N,      -
-  //                                                               - GemmBBlockCopyThreadClusterLengths_GemmK_GemmN in GridwiseConvolutionImplicitGemm_v4r4_nchw_kcyx_nkhw
-  //                                                               - Sequence<GemmBBlockCopyClusterLengths_GemmK, GemmBBlockCopyClusterLengths_GemmN>
-  //
-  //                                                                 - GemmBBlockCopyClusterLengths_GemmK =
-  //                                                                   k_per_block / ((n_per_block * k_per_lbock / block_size) / matrix_b_source_data_per_read)
-  //
-  //                                                                 - GemmBBlockCopyClusterLengths_GemmN =
-  //                                                                   n_per_block / matrix_b_source_data_per_read
-  //
-  //           typename BBlockCopyThreadClusterArrangeOrder,     - Sequence<0, 1>
-  //           typename BBlockCopySrcAccessOrder,                - Sequence<0, 1>
-  //           index_t BBlockCopySrcVectorReadDim,               - matrix_b_source_dest_vector_read_write_dim attribute
-  //           index_t BBlockCopySrcDataPerRead,                 - matrix_b_source_data_per_read attribute
-  //           index_t BBlockCopyDstDataPerWrite_N,              - matrix_b_dest_data_per_write_dim_n attribute
-  //           typename CThreadCopySrcDstAccessOrder,            - Sequence<0, 1, 2, 3>
-  //           index_t CThreadCopySrcDstVectorReadWriteDim,      - matrix_c_source_dest_vector_read_write_dim attribute
-  //           index_t CThreadCopyDstDataPerWrite>               - matrix_c_dest_data_per_write attribute
-
   func.walk([&](miopen::GridwiseGemmOp op) { affixTuningParametersImpl(op); });
 
   func.walk(
       [&](miopen::GridwiseGemmV2Op op) { affixTuningParametersImpl(op); });
```

Comment:
Looks like we can remove those because we'll never encouter a gridwisegemm op in this pass.

Reply:
In the next iteration they'll be removed.